ckan/templates/macros/form/dataset_errors.html

```diff
@@ -0,0 +1,51 @@
+{# (canada fork only): handle all errors in resource actions #}
+{# TODO: upstream contrib?? #}
+
+{#
+Builds a list of errors for the current form.
+
+errors  - A dict of field/message pairs.
+type    - The alert-* class that should be applied (default: "error")
+classes - A list of classes to apply to the wrapper (default: [])
+
+Example:
+
+{% import 'macros/form.html' as form %}
+{{ form.errors(error_summary, type="warning") }}
+
+#}
+
+
+{# (canada fork only): error -> danger BS version #}
+{% macro dataset_errors(errors={}, type="danger", classes=[], pkg_name='', helpers={}) %}
```

Comment:
rename to something more like resource_validation_errors to align with the python function?